library/src/test/java/com/ibm/research/drl/dpt/providers/identifiers/InternationalPhoneIdentifierTest.java

```diff
@@ -34,6 +34,13 @@ public void testIsOfThisType() {
         InternationalPhoneIdentifier identifier = new InternationalPhoneIdentifier();
 
         String[] validNumbers = {
+                "0044 0876653255",
+                "00 1 340 1234567",
+                "011 44 340 1234567",
+                "00 44-1481 1234567",
+                "+44-1481 1234567",
+                "+44 1481 1234567",
+                "00 1-441 1234567",
```

Comment:
I would not drop them, but move them to a different test, they are very use case specific tests but the identifier should still somehow cover them.